client/my-sites/plugins/categories/use-categories.tsx

```diff
@@ -66,13 +76,17 @@ export function useCategories(
 		ecommerce: {
 			name: __( 'Ecommerce & Business' ),
 			description: __( 'Ecommerce' ),
+			categoryDescription: __(
+				'Everything you need to turn your WordPress site into a powerful online store.'
+			),
 			icon: 'grid',
 			slug: 'ecommerce',
 			tags: [ 'ecommerce', 'e-commerce', 'woocommerce', 'business', 'business-directory' ],
 		},
 		booking: {
 			name: __( 'Booking & Scheduling' ),
 			description: __( 'Booking' ),
+			categoryDescription: __( 'Add a fully functionality booking system to your site.' ),
```

Comment:
@candy02058912 It looks like there's a typo here ([pointed out by a bot here](https://github.com/Automattic/wp-calypso/pull/66646#issuecomment-1220717277)), can we fix that before merging? Should be "Add a fully functional booking system to your site."

Reply:
Updated